cmd/neofs-node/config.go

```diff
@@ -860,6 +875,10 @@ func (c *cfg) configWatcher(ctx context.Context) {
 
 			c.shared.policer.Reload(c.policerOpts()...)
 
+			// Pool
+
+			c.reloadObjectPoolSizes()
```

Comment:
can you, please, put it before `// Policer` or even `// Logger`? imo it would be more convenient: first some internal thins and then a series of `Reload`